src/main/java/seedu/address/logic/parser/EditSupplierCommandParser.java

```diff
@@ -93,26 +95,30 @@ public EditSupplierCommand parse(String args) throws ParseException {
      * @throws ParseException Indicates the invalid format that users might have entered.
```

Comment:
same comment here as earlier; might want to rephrase for standardisation purposes